processing/src/main/java/org/apache/druid/segment/nested/ScalarDoubleColumn.java

```diff
@@ -50,18 +53,24 @@ public class ScalarDoubleColumn implements NestedCommonFormatColumn
       SemanticUtils.makeAsMap(ScalarDoubleColumn.class);
 
   private final FixedIndexed<Double> doubleDictionary;
+  private final Supplier<ColumnarInts> encodedValuesSupplier;
   private final ColumnarDoubles valueColumn;
-  private final ImmutableBitmap nullValueIndex;
+  private final ImmutableBitmap nullValueBitmap;
```

Comment:
Since it's a bitmap, this seemed cleaner. If there's a reason it's called index, we don't need to rename this.

Reply:
i named them `nullValueIndex` because this bitmap comes from the bitmap value indexes of the column, the selectors only need the null value index though so that they can satisfy the `isNull` method of the numeric selectors. Its not a big deal to rename them i guess, just calling it what it did sort of reflected where the bitmap came from

Reply:
That makes sense, I'll revert that to keep this context.